rclpy/rclpy/node.py

```diff
@@ -92,16 +92,20 @@
 
 
 class Node:
-
-    PARAM_REL_TOL = 1e-6
-
     """
     A Node in the ROS graph.
 
     A Node is the primary entrypoint in a ROS system for communication.
     It can be used to create ROS entities such as publishers, subscribers, services, etc.
     """
 
+    PARAM_REL_TOL = 1e-6
+    """
+    Minimum change in parameter value to register a change.
+
+    Changes smaller than this value will be ignored.
```

Comment:
@dawonn-haval this isn't exactly true. I see `PARAM_REL_TOL` being used to tell if a parameter value to which a floating point range constraint applies meets the constraint or not.

Reply:
@dawonn-haval Friendly ping.

Reply:
This totally dropped of my radar, sorry about the delay. He's totally right, I misunderstood the logic.

Updated the comment and rebased.